### ISSUE TITLE
rr_rover_zero_driver missing rospy dependency

### ISSUE THREAD

**Issue by tfoote** [RoverRobotics/rr_openrover_stack#80]
rr_rover_zero_driver is missing a dependency on rospy 

This is failing debian package builds 

http://build.ros.org/view/Kbin_uX64/job/Kbin_uX64__rr_rover_zero_driver__ubuntu_xenial_amd64__binary/79/console

```
16:18:49 -- Could not find the required component 'rospy'. The following CMake error indicates that you either need to install the package with the same name or change your environment so that it can be found.
16:18:49 CMake Error at /opt/ros/kinetic/share/catkin/cmake/catkinConfig.cmake:83 (find_package):
16:18:49   Could not find a package configuration file provided by "rospy" with any of
16:18:49   the following names:
16:18:49 
16:18:49     rospyConfig.cmake
16:18:49     rospy-config.cmake
16:18:49 
16:18:49   Add the installation prefix of "rospy" to CMAKE_PREFIX_PATH or set
16:18:49   "rospy_DIR" to a directory containing one of the above files.  If "rospy"
16:18:49   provides a separate development package or SDK, be sure it has been
16:18:49   installed.
16:18:49 Call Stack (most recent call first):
16:18:49   CMakeLists.txt:4 (find_package)
```

I see that the CircleCI jobs are failing too, but I can't seem to get public access to the results to review.

It's also an issue for the melodic branch: http://build.ros.org/job/Mbin_uB64__rr_rover_zero_driver__ubuntu_bionic_amd64__binary/

This is causing a regression of rr_openrover_stack which the new release depends on this new package.

**Comment by tfoote** [comment]
This is the one regression in the upcoming sync here: https://discourse.ros.org/t/preparing-for-kinetic-sync-2020-06-10/14614

**Comment by padiln** [comment]
Dependency issues appear to be resolved in latest release 1.0.1 and 1.1.1.

Apologies for any disruptions.
